src/Tribe/Admin/Notice/Timezones.php

```diff
@@ -67,17 +138,20 @@ public function notice() {
 			return false;
 		}
 
-		$url = 'https://theeventscalendar.com/knowledgebase/time-zones/';
+		$text = [];
+		$current_utc = Timezones::wp_timezone_string();
+
+		$url = 'http://m.tri.be/1acz';
 		$link = sprintf(
 			'<a href="%1$s" target="_blank">%2$s</a>',
 			esc_url( $url ),
 			esc_html__( 'Read more', 'the-events-calendar' )
 		);
-		$text = __( 'When using The Events Calendar, we recommend that you choose a city in your timezone and avoid using a UTC timezone offset. Choosing a UTC timezone may cause problems when importing events or with Day Light Savings time. %1$s', 'the-events-calendar' );
 
-		return sprintf( $text, $link );
-
-	}
+		$text[] = __( 'When using The Events Calendar, we recommend that you geographic timezone such as "America/Los Angeles" and avoid using a UTC timezone offset such as “%2$s”.', 'the-events-calendar' );
```

Comment:
should be `we recommend that you use a geographic timezone such as "America/Los_Angeles"`

Reply:
hey @bordoni, you changed the time zone string in the notice which is correct, but you also still have to fix `we recommend that you geographic`

this should be `we recommend that you use a geographic`